python/ql/src/Security/CWE-295/RequestWithoutValidation.qhelp

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Encryption is key to the security of most, if not all, online communication.
+Using TLS can enusre that neither party in the communication is an interloper.
+For this reason, is is unwise to disable the verification that TLS provides.
+<code>requests</code> provides verification by default, and it is only when
```

Comment:
Suggest: "The `requests` method provides verification by default. To disable TLS verification you have to explicitly turn it off using `verify=False`." - assuming that `requests` is a method. It should read better and avoids starting a sentence with lowercase.

Reply:
`requests` is a module containing several functions, all of which have the optional `verify` parameter.